src/helpers.js

```diff
@@ -35,3 +36,12 @@ export const execShellCommand = (cmd) => {
     });
   });
 }
+
+export const getValidatedInput = (key) => {
+	const regex = new RegExp('/^[-.+A-Za-z0-9]*$/')
+  const value = core.getInput(key);
+  if (regex.test(value)) {
+    throw new Error(`Invalid value for '${key}': '${value}'`);
+  }
+  return value;
```

Comment:
```suggestion
  const value = core.getInput(key);
  if (/^[-.+A-Za-z0-9]*$/.test(value)) {
    throw new Error(`Invalid value for '${key}': '${value}'`);
  }
  return value;
```
re-using a regexp object in JavaScript is dangerous (Regexp.lastIndex needs to set to 0 otherwise if its /g)

Reply:
Thank you! I committed you suggestion :)